staticfiles/templates/authentication/country.html

```diff
@@ -0,0 +1,242 @@
+<select>
```

Comment:
I will delete this html. But, shouldn't the data be read from the json file(country.json) since this data (for country names ) will not change. Please let me know where am i wrong, i will rectify it.